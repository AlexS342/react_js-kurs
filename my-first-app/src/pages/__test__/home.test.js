import { render } from '@testing-library/react';
import Home from '../home';

it('Снимок страници Home', () => {
    const component = render(<Home />);
    expect(component).toMatchSnapshot();
})