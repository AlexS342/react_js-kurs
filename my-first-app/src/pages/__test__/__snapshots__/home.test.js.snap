// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Снимок страници Home 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="home"
      >
        <h1
          class="home-title"
        >
          Добро пожаловать
        </h1>
        <img
          alt="logo"
          class="home-img"
          src="logo192.png"
        />
        <h2
          class="home-subTitle"
        >
          Мы используем React
        </h2>
        <p
          class="home-p"
        >
          Это главная страница
        </p>
        <p
          class="home-p"
        >
          По заданию проекта к этой странице не предьявлено ни каких требований и вероятней всего, она не будет наполнятся ни какой информацией.
        </p>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="home"
    >
      <h1
        class="home-title"
      >
        Добро пожаловать
      </h1>
      <img
        alt="logo"
        class="home-img"
        src="logo192.png"
      />
      <h2
        class="home-subTitle"
      >
        Мы используем React
      </h2>
      <p
        class="home-p"
      >
        Это главная страница
      </p>
      <p
        class="home-p"
      >
        По заданию проекта к этой странице не предьявлено ни каких требований и вероятней всего, она не будет наполнятся ни какой информацией.
      </p>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
